compiler/src/pretyping.ml

```diff
@@ -81,7 +81,8 @@ let pp_suffix fmt =
   | PVx (szo, szi) -> F.fprintf fmt "_u%au%a" pp_wsize szo pp_wsize szi
   | PVvv (ve, sz, ve', sz') -> F.fprintf fmt "_%s_%s" (string_of_velem Unsigned sz ve) (string_of_velem Unsigned sz' ve')
 
-let pp_tyerror fmt (code : tyerror) =
+let pp_tyerror ~debug fmt (code : tyerror) =
```

Comment:
Is it ever useful to call this function with `debug:true`? IMHO these error messages hardly need the extra verbosity.